opengever/maintenance/scripts/bumblebee_installation.py

```diff
@@ -130,12 +130,21 @@ def main(app, argv=sys.argv[1:]):
         brains = catalog.unrestrictedSearchResults(
             {'object_provides': 'ftw.bumblebee.interfaces.IBumblebeeable'})
 
+        skipped = []
         for index, brain in enumerate(ProgressLogger(
                 'Create checksums for objects in portal repository', brains,
                 logger=LOG)):
 
             obj = brain.getObject()
             versions = repository.getHistory(obj)
+
+            if len(versions) > 50:
+                LOG.info(
+                    'Skipping object with more than 50 versions: %s' % obj.id)
+                skipped.append((obj.id, obj.absolute_url()))
+                continue
+
+            LOG.info('  Calculating version checksums for %s' % obj.id)
```

Comment:
Indented by two spaces to make it easier to distinguish the per-object logging (which can get quite noisy) from the per-batch logging.